### PR TITLE
add sqlalchemy-spanner

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hadim** [comment]
@conda-forge/staged-recipes, ready for review

CI has not been triggered. Is it an error or a new behaviour?

**Comment by hadim** [comment]
Once the lib is on pypi I will switch the source to it. See https://github.com/cloudspannerecosystem/python-spanner-sqlalchemy/issues/58

**Comment by hadim** [comment]
Link to CI https://dev.azure.com/conda-forge/feedstock-builds/_build?definitionId=2259&_a=summary

**Comment by hadim** [comment]
CI is green: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=365967&view=results

@conda-forge/staged-recipes, ready for review

File: recipes/sqlalchemy-spanner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "sqlalchemy-spanner" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/cloudspannerecosystem/python-spanner-sqlalchemy/archive/refs/tags/v0.1.0.zip
```
Comment (xhochy) [inline recipes/sqlalchemy-spanner/meta.yaml:10]:
```suggestion
  url: https://github.com/cloudspannerecosystem/python-spanner-sqlalchemy/archive/refs/tags/v{{ version }}.zip
```

Comment (hadim) [reply recipes/sqlalchemy-spanner/meta.yaml:10]: Thanks for the catch!

**Comment by hadim** [comment]
CI is green https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=365976&view=results

**Comment by hadim** [comment]
Thank you for the review @xhochy !

**Pull request merged**